pkg/transforms/coredata.go

```diff
@@ -22,3 +27,20 @@ func (cdc *CoreData) MarkAsPushed(edgexcontext *appcontext.Context, params ...in
 	}
 	return true, params[0]
 }
+
+// PushToCoreData pushes the provided value as an event to CoreData using the device name and device resource name that have been set.
+func (cdc *CoreData) PushToCoreData(edgexcontext *appcontext.Context, params ...interface{}) (bool, interface{}) {
+	if len(params) < 1 {
+		// We didn't receive a result
+		return false, errors.New("No Data Received")
+	}
+	val, err := util.CoerceType(params[0])
+	if err != nil {
+		return false, err.Error()
+	}
+	result, err := edgexcontext.PushtoCoreData(cdc.DeviceName, cdc.ReadingName, val)
+	if err != nil {
+		return false, err.Error()
+	}
+	return true, result
```

Comment:
Currently this will return the new Event's ID to the next function, should be the new Event?

Reply:
Ahh, need to also set it in the Context...